index.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./src/client/assets/output.css" />
```

Comment:
How is the CSS file included if you leave it out from HTML?

Reply:
It will be inlined into the output since you are using the singlefile vite plugin.

Reply:
OK